src/trace-span.ts

```diff
@@ -36,7 +36,7 @@ export class TraceSpan {
   constructor(
     public readonly name: string,
     public readonly spanId: number,
-    public readonly parentSpanId: number
+    public readonly parentSpanId: number | string
```

Comment:
From the rest of the changes, I cannot see why this definition needed to change. What's the reason here?

Reply:
https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/567/files#diff-4fae65ee00e417727e40216b35f1cbe7R221

Incoming span ID is a UInt64, so AFAIK we need it to be a string to represent it exactly.

Reply:
Would it make sense for it to be only a `string` then? AFAIK we don't need to do arithmetic on it.

Reply:
Yep -- but to keep things consistent I'll do likewise for other references to span IDs.